src/MfaVerifyController.js

```diff
@@ -114,6 +118,9 @@ function (Okta, Checkbox, BaseLoginController, CookieUtil, TOTPForm, YubikeyForm
     },
 
     trapAuthResponse: function () {
+      if(this.options.factorType === 'password') {
```

Comment:
will this still be needed once the backend returns ```MFA_REQUIRED```?

Reply:
Yea, if it's MFA_REQUIRED it will trap the response and will not move on to the normal mfa flow

Reply:
can you explain this to me a little more? I think that if we get ```MFA_REQUIRED``` then it is just like any other factor, so steps are something like

1. bootstrap with a stateToken
2. refreshAuthSate with that stateToken -> returns MFA_REQUIRED
3. RouterUtil navigate to MFA_REQUIRED view, with the first factor on the list, which is password
4. Type password and verify -> response is MFA_REQUIRED again with a different list of factors
5. RouterUtil navigates again to MFA_REQUIRED view, but with a different factor

Is this the flow?

Reply:
got it. Makes sense... guess it goes back to the initial talk we have about removing ```isMfaRequired``` from the ```if``` block here... which will make this work for more than 1 mfa_required responses.
But I remember there was a problem with that approach too, right?

Reply:
Yea it causes a problem when a factor is in mfa challenge and you switch to another factor in mfa required

Reply:
We had an offline conversation about this. This is good and will work perfect for password re-auth, and we will file a jira to make this generic enough to support chained MFA_REQUIRED responses for any factor.